### PR TITLE
[feat] multimodal inputs for reward model

### PR DESCRIPTION
**Pull request by mingruimingrui**
Reward model was missing multimodal inputs. It's already been added for the actor & critic workers.

### CODE REVIEW

File: verl/workers/fsdp_workers.py [hunk 1]
```diff
@@ -1025,6 +1025,14 @@ def _forward_micro_batch(self, micro_batch):
             attention_mask = micro_batch['attention_mask']
             position_ids = micro_batch['position_ids']
 
+            multi_modal_inputs = {}
```
Comment (hiyouga) [inline verl/workers/fsdp_workers.py:1028]: It is unused actually

**Review by hiyouga — CHANGES_REQUESTED** [review]
See the above comment

File: verl/workers/fsdp_workers.py (hunk 1 above)
Comment (mingruimingrui) [reply verl/workers/fsdp_workers.py:1028]: My bad! Fixed

**Review by hiyouga — APPROVED** [review]
